cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -1288,6 +1289,10 @@ def _add_head_node(self):
                     "efs_iam_authorizations": to_comma_separated_string(
                         self.shared_storage_attributes[SharedStorageType.EFS]["IamAuthorizations"], use_lower_case=True
                     ),
+                    "efs_accesspoint_ids": to_comma_separated_string(
```

Comment:
This change should be captured in unit test here: https://github.com/aws/aws-parallelcluster/blob/develop/cli/tests/pcluster/templates/test_cluster_stack.py#L901